PendingReleaseNotes

```diff
@@ -74,38 +74,36 @@
   be enabled to migrate to the new format. See
   https://docs.ceph.com/en/squid/radosgw/zone-features for details. The "v1"
   format is now considered deprecated and may be removed after 2 major releases.
-* CEPHFS: MDS evicts clients which are not advancing their request tids which causes
-  a large buildup of session metadata resulting in the MDS going read-only due to
-  the RADOS operation exceeding the size threshold. `mds_session_metadata_threshold`
-  config controls the maximum size that a (encoded) session metadata can grow.
+* CephFS: The MDS evicts clients which are not advancing their request tids, which causes
+  a large buildup of session metadata, which in turn results in the MDS going read-only
+  due to RADOS operations exceeding the size threshold. `mds_session_metadata_threshold`
+  config controls the maximum size to which (encoded) session metadata can grow.
 * CephFS: A new "mds last-seen" command is available for querying the last time
   an MDS was in the FSMap, subject to a pruning threshold.
-* CephFS: For clusters with multiple CephFS file systems, all the snap-schedule
+* CephFS: For clusters with multiple CephFS file systems, all snap-schedule
   commands now expect the '--fs' argument.
 * CephFS: The period specifier ``m`` now implies minutes and the period specifier
-  ``M`` now implies months. This has been made consistent with the rest
-  of the system.
+  ``M`` now implies months. This is consistent with the rest of the system.
 * RGW: New tools have been added to radosgw-admin for identifying and
   correcting issues with versioned bucket indexes. Historical bugs with the
   versioned bucket index transaction workflow made it possible for the index
   to accumulate extraneous "book-keeping" olh entries and plain placeholder
   entries. In some specific scenarios where clients made concurrent requests
-  referencing the same object key, it was likely that a lot of extra index
+  referencing the same object key, it was likely that extra index
   entries would accumulate. When a significant number of these entries are
   present in a single bucket index shard, they can cause high bucket listing
-  latencies and lifecycle processing failures. To check whether a versioned
+  latency and lifecycle processing failures. To check whether a versioned
   bucket has unnecessary olh entries, users can now run ``radosgw-admin
   bucket check olh``. If the ``--fix`` flag is used, the extra entries will
-  be safely removed. A distinct issue from the one described thus far, it is
-  also possible that some versioned buckets are maintaining extra unlinked
-  objects that are not listable from the S3/ Swift APIs. These extra objects
-  are typically a result of PUT requests that exited abnormally, in the middle
-  of a bucket index transaction - so the client would not have received a
+  be safely removed. An additional issue  is that some versioned buckets are
+  may maintain extra unlinked
```

Comment:
There's one too many verbs in this sentence. "An additional issue is that some versioned buckets are may maintained extra unlimited objects that are not listable via the S3/ Swift APIs." should probably be "An additional issue is that some versioned buckets may maintain extra unlinked objects that are not listable via the S3/Swift APIs."